ct.sh

```diff
@@ -43,13 +43,16 @@ if [ "$TRAVIS_RUST_VERSION" == "stable" ] || [ "$TRAVIS_RUST_VERSION" == "beta"
         if [ -n "$AES_NI_SUPPORT" ]; then
             cargo test --features force_aesni_support --target $TARGET
         fi
+
+        # no_std tests only be able to run on x86 platform
```

Comment:
Minor: The comment should probably say "no_std tests are only able" instead of "no_std tests only be able"

Reply:
thx for point out
fixed